romankh3/image-comparison
# Implement simpleComparison() and create load tests for issue #119 and #174

# PR Details


- Provide another kind of pattern ```simpleComparison()```, which makes it faster than ```compareImages()```.
- Creare load tests to see if ```simpleComparison()``` method is faster.
## Description


- Implement a method called  ```simpleComparison()```, which makes it faster to compare images without drawing rectangles. 
- Add ```isFirstDifference()``` method to judge the first different pixel of two images.
- Create load tests to verify time differences between ```simpleComparison()``` and ```compareImages()```.
## Related Issue





- issue #119 
- issue #174 
## Motivation and Context


The change is aim to users who don't want to know what differences of compare images. Some users only need to know that if two images are different or not. This ```simpleComparison``` pattern can makes it faster for users to get result quickly.

## How Has This Been Tested




- Add a new test file  called _LoadTest.java_, in which I designed some load test to test speed differernces.
- Verify the ```simpleComparison()``` feature to make sure it could run normally. Check out the compare result ```ComparisonState```.
- For some image instances, calculate running time of ```simpleComparison()``` and ```compareImages()``` method. ```simpleComparison()``` should be faster than ```compareImages()```.
## Types of changes




## Checklist
